doc/tutorials/Chapter_1/01-Setup_Installation.md

```diff
@@ -0,0 +1,302 @@
+# ROHD Setup & Installation
+
+There are two options for developing with ROHD, either through running it on a GitHub Codespace or on your local machine.
+
+
+## Setup on Github Codespaces (Recommended)
+
+### Step 1: Click on the CodeSpace Button
+
+To access the Codespaces feature on the https://github.com/intel/rohd repository, simply click on the "Codespaces" button. 
+
+Please note that Codespaces are free for all users with a personal GitHub account that have either a Free or Pro plan. However, there is a monthly usage limit. To learn more about this feature, visit the official GitHub Codespaces Overview page at https://docs.github.com/en/codespaces/overview.
+
+> All personal GitHub.com accounts have a monthly quota of free use of GitHub Codespaces included in the Free or Pro plan. You can get started using GitHub Codespaces on your personal account without changing any settings or providing payment details. You can create and use a codespace for any repository you can clone. You can also use a template to create codespaces that are not initially associated with a repository. If you create a codespace from an organization-owned repository, use of the codespace will either be charged to the organization (if the organization is configured for this), or to your personal account. Codespaces created from templates are always charged to your personal account. You can continue using GitHub Codespaces beyond your monthly included storage and compute usage by providing payment details and setting a spending limit. For more information, see "About billing for GitHub Codespaces.
+
+![step 1](assets/CodespaceSetup/step1.PNG)
+
+### Step 2: Build Container
+
+You will be redirected to a page where GitHub will initiate the launch of your container. Please allow some time for GitHub to establish your server.
+
+![step 2](assets/CodespaceSetup/step2.PNG)
+
+### Step 3: VSCode in Browser
+
+When your space is ready, you will see a visual studio code running on your browser.
+
+![step 3](assets/CodespaceSetup/step3.PNG)
+
+### Step 4: Dart pub get
+
+Run `dart pub get` on the terminal of the visual studio code to pull your setup.
+
+![step 4](assets/CodespaceSetup/step4.PNG)
+
+### Step 5: Run the example code 
+
+Open up `example` folder on the left navigation panel and click on `example.dart` to bring forward the first example of ROHD. After that, navigate to the main function at below of line 58 and click on the `Run` at `Run | Debug`.
+
+![step 5](assets/CodespaceSetup/step5.PNG)
+
+
+If you can see SystemVerilog code pop up on the terminal. Well, you have successfully set up your development environment on the cloud.
+
+### Step 6: Delete the CodeSpace
+
+To delete the codespace, go back to https://github.com/intel/rohd and click on codespace just like step 1. But this time, you will see more options. Click on the `delete` option to delete codespace.
```

Comment:
While I don't think we need to get to 100% for this PR, try to check for consistency in names, spelling, and capitalization.  For example, "ROHD", "SystemVerilog", "Codespace", etc. should always be spelled and capitalized the same.